### DISCUSSION TITLE
FeatureFinderMultiplex Error

### DISCUSSION THREAD

**Discussion by acragin** [OpenMS/OpenMS#6617] [Q&A]
Hello,

I am trying to run a file through the MetaProSIP openMS KNIME pipeline however have run into an issue early on where my mzML file will experience and error when trying to read it with FeatureFinderMultiplex. The error reads:
Error: Unable to read file (- due to that error of type Parse Error in: /Users/builder/jenkins/ws/openms/RC/openms_release_packaging/87227eb0/source/src/openms/source/FORMAT/HANDLERS/XMLHandler.cpp@131-void OpenMS::Internal::XMLHandler::fatalError(OpenMS::Internal::XMLHandler::ActionMode, const OpenMS::String &, OpenMS::UInt, OpenMS::UInt) const)]

Anyone have any insights on why this might be happening or how to resolve?

Thanks!

**Comment by jpfeuffer** [comment]
This usually really is an error in you file.
Where did you get it from (download corrupt)? Was it preprocessed? Where is it located (weird shared drive?)? Do you have enough space?
Windows, Mac, Linux?

Try to run FileInfo or MzMLValidator. They will probably fail, too.
